engine/security/apparmor.md

```diff
@@ -16,9 +16,13 @@ operating system and its applications from security threats. To use it, a system
 administrator associates an AppArmor security profile with each program. Docker
 expects to find an AppArmor policy loaded and enforced.
 
-Docker automatically loads container profiles. The Docker binary installs
-a `docker-default` profile in the `/etc/apparmor.d/docker` file. This profile
-is used on containers, _not_ on the Docker Daemon.
+Docker automatically generates and loads a default profile for containers that's
+named `docker-default`. On Docker versions `1.13.0` and later, the Docker binary will
+generate this profile in `tmpfs` and then load it into the kernel. On Docker versions
```

Comment:
Prefer present tense (generates, is generated) to future tense (will generate, will be generated)